repositories/UserRepository.ts

```diff
@@ -48,9 +48,9 @@ export class UserRepository extends BaseRepository<UserModel> {
   public async getAllEmails(): Promise<string[]> {
     const emailsRaw = await this.repository
```

Comment:
can we change this variable name to `emailsNamesRaw` so it's a bit more descriptive?